tests/console/yast2_ftp.pm

```diff
@@ -14,8 +14,59 @@ use strict;
 use base "console_yasttest";
 use testapi;
 use utils;
+use version_utils;
+use y2logsstep;
+
+sub vsftd_setup_checker {
+    my $config_ref              = pop();
+    my $error                   = undef;
```

Comment:
We can init it as empty line, or we simply declare it, as it will be undef and we don't need to set it explicitly. Otherwise LGTM.